.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -46,3 +55,25 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           event-type: build-images
           client-payload: '{"ref": "refs/heads/staging"}'
+
+      - name: Send slack notification if main not merged into staging
+        if: failure()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          fields: repo,commit,author,eventName,workflow,job,mention
+          mention: "here"
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+  notify_on_test_failure_on_main:
```

Comment:
Will this be redundant with notifications already sent by push-tests.yml?

Reply:
Good catch! Will fix!